frontend/store/collaterals.ts

```diff
@@ -56,15 +71,21 @@ export const actions = {
                 marketUnitPrice,
                 secondsBetweenPriceDrops: calcParameters.secondsBetweenPriceDrops,
                 priceDropRatio: calcParameters.priceDropRatio,
+                uniswap: {
+                    type: 'token',
+                    route: tokenAddress,
+                },
```

Comment:
As specified above, please don't reuse the same `uniswap` object as used in the collateral config, that is very confusing